Rakefile

```diff
@@ -24,7 +26,21 @@ task :build => [:dependencies, :submodules] do
   sh('bundle exec jekyll build --config _config.yml')
 end
 
-task :validate => [:build] do
+task :validate_kit_versions do
+  data = YAML.load_file('_data/kit_versions.yml')
+  data.each do |entry|
+    actual = entry.keys.to_set
+    expected = ['version', 'released'].to_set
+    missing = expected - actual
+    extra = actual - expected
+    messages = []
+    messages << "Missing keys: #{missing.to_a.join(', ')}" if missing.size() > 0
+    messages << "Extra keys: #{extra.to_a.join(', ')}" if extra.size() > 0
+    raise "For entry\n#{entry}\n#{messages.join("\n")}\n\n" if messages.size() > 0
```

Comment:
Is it worth validating the date format, and order of dates for the versions too?

Reply:
Maybe? I wonder if those would more easily be solved by a comment in the file?
The goal here was for this to be a quick sanity check rather than being exhaustive.